x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-config-topics.adoc

```diff
@@ -108,23 +108,23 @@ These control the types of writer operations.
-|sendUniqueTransactionNumbers="false" |Send unique Transaction Set Control Numbers. `False` (default) assigns transaction set control numbers sequentially within each functional group and reuses them in different groups. `True` assigns unique transaction set numbers across all interchanges sent to the same partner and application. It is derived by the interchange sender and receiver identification, combined with the functional group sender and receiver application codes
-|implementationConventionReference="" |Implementation convention reference for transactions (ST segment). Setting this value uses it as the ST Implementation Convention Reference unless overridden in the message parameters
-|initialInterchangeNumber="1" |The initial Interchange Control Number used for outgoing messages
-|initialGroupNumber="1" |The initial Group Control Number used for outgoing messages
-|initialSetNumber="1" |The initial Transaction Set Control Number used for outgoing messages
+|sendUniqueTransactionNumbers="false" |Send unique Transaction Set Control Numbers. `False` (default) assigns transaction set control numbers sequentially within each functional group and reuses them in different groups. `True` assigns unique transaction set numbers across all interchanges sent to the same partner and application. It is derived by the interchange sender and receiver identification, combined with the functional group sender and receiver application codes.
+|implementationConventionReference="" |Implementation convention reference for transactions (ST segment). Setting this value uses it as the ST Implementation Convention Reference unless overridden in the message parameters.
+|initialInterchangeNumber="1" |The initial Interchange Control Number used for outgoing messages.
+|initialGroupNumber="1" |The initial Group Control Number used for outgoing messages.
+|initialSetNumber="1" |The initial Transaction Set Control Number used for outgoing messages.
 |ackRequested="false" |Request acknowledgments for sent transactions flag. If true, `997` or `999` acknowledgments are requested for all sent transactions.
-|defaultUsageIndicator="P" |Default ISA15 interchange usage indicator: `I` for Information, `P` for Production Data, `T` for Test Data
+|defaultUsageIndicator="P" |Default ISA15 interchange usage indicator: `I` for Information, `P` for Production Data, `T` for Test Data.
 |useSuppliedValues="false" |Use values from supplied data for control segment identifiers (ISA/IEA, GS/GE, ST/SE segments). The default is always to generate control numbers when writing allowing you to use chosen values.
```

Comment:
This is really unclear: the value here is false, but the description is true, to use values. And the default sounds like it is true as well, although there is at least one clarifying comma missing.
Reword the end to say "Setting this value so that a control number is always generated during a write operation enables you to"...use "chosen" values? Which chosen values? chosen by what/whom?